src/Infrastructure.Windows/SoftwareManagers/OneGameLauncherManager.cs

```diff
@@ -0,0 +1,51 @@
+﻿namespace AutoGame.Infrastructure.Windows.SoftwareManagers;
+
+using System;
+using System.IO;
+using System.IO.Abstractions;
+using AutoGame.Core.Interfaces;
+using AutoGame.Infrastructure.Windows.Interfaces;
+using Serilog;
+
+internal sealed class OneGameLauncherManager : ISoftwareManager
+{
+    public OneGameLauncherManager(
+        ILogger logger,
+        IUser32Service user32Service,
+        IFileSystem fileSystem,
+        IProcessService processService,
+        IRegistryService registryService)
+    {
+        this.Logger = logger;
+        this.User32Service = user32Service;
+        this.FileSystem = fileSystem;
+        this.ProcessService = processService;
+        this.RegistryService = registryService;
+    }
+
+    private ILogger Logger { get; }
+    private IUser32Service User32Service { get; }
+    private IFileSystem FileSystem { get; }
+    private IProcessService ProcessService { get; }
+    private IRegistryService RegistryService { get; }
```

Comment:
Looks like you can get rid of the `ILogger`, `IFileSystem`, and `IRegistryService` dependencies since they aren't used.